task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,8 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    return num % 3 === 0 && num % 5 === 0 ?
```

Comment:
In this case, the implementation via "if" "return" is much better.

Reply:
refactor to if - else if contruction for better readabitilty